packages/react/src/utils/index.ts

```diff
@@ -1,3 +1,5 @@
+export { assert } from './assert'
 export { hasResetKeysChanged } from './hasResetKeysChanged'
 export { hashKey } from './hashKey'
-export { assert } from './assert'
+export { isClient } from './isClient'
+export { isServer } from './isServer'
```

Comment:
I think we don't need to make unnecessary function signature now. if there is needs to make it after, Let's make it then.

```suggestion
```